### DISCUSSION TITLE
How to run :terminal {command} without a new buffer popping up

### DISCUSSION THREAD

**Discussion by idelice** [LazyVim/LazyVim#2517] [General]
I want to run a formatting cmd `mvn fmt:format` when working on maven projects, and I wasn't sure how to do it through `conform` (if that is even possible?), so I normally execute it by running `:terminal {command}`. 

The annoyance is that a new buffer shows up which I don't want.

Is there a better way to approach this?

**Comment by mikalcallahan** [comment]
You can use `<c-/>` to open/hide a floating terminal. That's what I do when I need to run some one-off commands

**Reply by idelice** [reply]
That's pretty neat. Didn't know we had such a keymap. Thank you

**Reply by kobzar** [reply]
https://www.lazyvim.org/keymaps
